### DISCUSSION TITLE
Experiments running but inputs and outputs of LLM are not linked to dashboard

### DISCUSSION THREAD

**Discussion by IbrahimAhmed03** [langfuse/langfuse#2590] [Support]
Hi, 

I am currently working on a project where I am automating testing using the langfuse datasets feature on a localhost. The current problem I am having is that when I am linking to my dashboard, the inputs and outputs that the model is working with is not showing up on the langfuse dashboard for me to see. 

Whenever I print the values out they show up in my terminal but just not in the langfuse dashboard. I will provide screenshots for reference. Is there a way I can link the item input and output to the dashboard so I can see the output the model gives in comparison with the expectedOutput for each item in the dataset? I am providing the code below and relevant screenshots. Any assistance would be appreciated. 

<img width="1565" alt="image" src="https://github.com/langfuse/langfuse/assets/105565549/59f5223d-147f-464d-ae57-6e1615aaf6b5">

<img width="512" alt="image" src="https://github.com/langfuse/langfuse/assets/105565549/801283cf-f365-4294-93de-f5eed2e5a271">

<img width="716" alt="image" src="https://github.com/langfuse/langfuse/assets/105565549/87d4649a-2429-46e9-8ec4-617b5fd4a1df">

**Comment by marcklingen** [comment]
The input/output shown in this table is the trace input/output. Do your traces that you create during this experiment have inputs and output values set? Can you share how you create traces during your experimentation runs?
